index.css

```
@@ -0,0 +1,86 @@
+body{
+    font-family: "Inter", sans-serif;
+    font-size: 20px;
+    padding: 0;
+    margin: 0;
+}
+h1{
+    font-size: 80px;
+    font-weight: 500;
+}
+h2{
+    font-size: 64px;
+    font-weight: 500;
+}
+h3{
+    font-size: 48px;
+    color: #fff;
+    text-align: center;
+    font-weight: 500;
+}
+h4{
+    font-size: 36px;
+    font-weight: 500;
+}
+.container{
+    width: 1380px;
+    margin: 0 auto; /*позиционирование по центру*/
+}
+.header{
+    background: url(pictures/banner1.jpg) no-repeat center top / cover;
+    height: 900px;
+}
+header a {
+    color: #fff;
+}
+.menu{
+    display: flex;
+ }
```

На будущее, чтобы все элементы под flex были в одну строчку, можешь сделать так:

```suggestion
.menu{
    display: flex;
    align-items: center;
    justify-content: center;
    gap: 40px;  /* по факту заменит здесь margin-right:40 px; /*
 }
```